### PR TITLE
Hotfix/Dev 3382 & 3680 Homepage Updates for CG & Total Spending

### PR DESCRIPTION
**Pull request by jonhill13**

### CODE REVIEW

File: src/js/components/homepage/features/PaneFeature.jsx [hunk 1]
```diff
@@ -4,36 +4,54 @@
...
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 const PaneFeature = () => (
     <div className="feature-pane">
         <div className="feature-pane__wrapper">
             <h2 className="feature-pane__title">Featured Content</h2>
             <div className="feature-pane__content-wrapper">
                 <div className="feature-pane__content">
-                    <h3 className="feature-pane__content-title">New! - Congressional Budget Justifications</h3>
+                    <div className="feature-pane__content-title">New! - Congressional Budget Justifications</div>
```
Comment (MaxwellKendall) [inline src/js/components/homepage/features/PaneFeature.jsx]: Lets keep this an <h3> in an effort to keep our html semantic

File: src/js/components/sharedComponents/header/InfoBanner.jsx [hunk 2]
```diff
@@ -26,37 +26,34 @@ export default class InfoBanner extends React.Component {
...
+                        <span className="info-banner__hyphen">-</span>
                         <a
-                            href="https://usaspending-help.zendesk.com/hc/en-us/community/topics"
+                            href="https://datalab.usaspending.gov/americas-finance-guide/"
                             target="_blank"
                             rel="noopener noreferrer"
-                            onClick={clickedBannerLink.bind(null, 'https://usaspending-help.zendesk.com/hc/en-us/community/topics')}>
-                            Community
-                        </a> page.
+                            onClick={clickedBannerLink.bind(null, 'https://datalab.usaspending.gov/americas-finance-guide/')}>
```
Comment (MaxwellKendall) [inline src/js/components/sharedComponents/header/InfoBanner.jsx]: lets define this in the constructor to avoid creating an anon func every render.

Comment (MaxwellKendall) [reply src/js/components/sharedComponents/header/InfoBanner.jsx]: Could also just be a const inside the render.

File: src/js/components/sharedComponents/header/InfoBanner.jsx [hunk 3]
```diff
@@ -26,37 +26,34 @@ export default class InfoBanner extends React.Component {
...
+                        </span>
                     </div>
                     <button
                         className="info-banner__close-button"
                         title="Dismiss message"
                         aria-label="Dismiss message"
                         onClick={this.bannerClosed}>
-                        <Close alt="Dismiss message" />
+                        <FontAwesomeIcon size="lg" alt="Dismiss message" icon="times" />
+                        {/* <Close alt="Dismiss message" /> */}
```
Comment (MaxwellKendall) [inline src/js/components/sharedComponents/header/InfoBanner.jsx]: Delete this?

File: src/js/components/homepage/features/PaneFeature.jsx [hunk 4]
```diff
@@ -13,27 +14,44 @@ const PaneFeature = () => (
...
                 <hr className="feature-pane__mobile-content-divider" />
-                <div className="feature-pane__content">
-                    <h3 className="feature-pane__content-title">New! – Your Guide to America’s Finances</h3>
-                    <p className="feature-pane_content-text">Your Guide presents straightforward information about the federal government’s spending and revenue, as well as the deficit and debt in 2018.</p>
-                    <div className="feature-pane__button-wrapper">
-                        <a href="https://datalab.usaspending.gov/americas-finance-guide/" target="_blank" rel="noopener noreferrer" className="feature-pane__button">Explore the Data</a>
+                <div className="feature-pane__content feature-pane__content-middle__sub">
+                    <div className="">
+                        <div id="feature-pane__content-header__middle">
+                            Updated for FY 2019
```
Comment (MaxwellKendall) [inline src/js/components/homepage/features/PaneFeature.jsx]: looks like this should be `feature-pane__content-middle__sub`

Comment (MaxwellKendall) [reply src/js/components/homepage/features/PaneFeature.jsx]: my bad, this comment should be on line 24

Comment (MaxwellKendall) [reply src/js/components/homepage/features/PaneFeature.jsx]: Cool, so, if there's no className, we should just not even use the className attribute.

Comment (jonhill13) [reply src/js/components/homepage/features/PaneFeature.jsx]: just pushed

**Pull request merged**